integrationtest/src/it/scala/io/gearpump/integrationtest/checklist/StormCompatibilitySpec.scala

```diff
@@ -19,52 +19,72 @@ package io.gearpump.integrationtest.checklist
 
 import io.gearpump.integrationtest.TestSpecBase
 import io.gearpump.integrationtest.minicluster.Util
+import io.gearpump.integrationtest.storm.StormClient
 
 /**
  * The test spec checks the compatibility of running Storm applications
  */
 class StormCompatibilitySpec extends TestSpecBase {
 
-  private val STORM_STARTER_JAR = "/opt/gearpump/lib/storm/storm-starter-0.9.5.jar"
+  private val stormClient = new StormClient(cluster, commandLineClient)
```

Comment:
It is already to late to override a cluster in this way. Cluster was started once in StandaloneModeSpec, before `beforeAll` here.